packages/openapi-tanstack-query/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "openapi-tanstack-query",
```

Comment:
I'll do that now. What do you think of `openapi-tanstack-react-query` ?
A bit longer but it lets the door open for Vue, Angular, Solid and Svelte

Reply:
Or `openapi-react-query` as `react-query` is old version of `@tanstack/react-query`, but for SEO it loses the tanstack word. Maybe a tag would be enough.
What do you think ?

Reply:
I like that. And yeah that makes sense because TanStack does support other frameworks.

As a sidenote, I own the `@oapi-ts` npm scope, but haven’t really used it for anything. `@openapi-ts`, unfortunately, is in use by other unrelated project, so I’ve been hesitant to adopt a scope that’s different from the GitHub org + domain name (`openapi-ts.dev`). Probably just unscoped packages makes the most sense at this point.

Reply:
When it starts to be more an ecosystem and not just a single library I prefer to have a scope, but that's really personal opinion and I don't think it makes much difference. So unscope is good for now I think. Maybe if it really starts to be an ecosystem an org scope will be interesting.

According to Google, not a lot of people search for tanstack, so it will be `openapi-react-query`.
![image](https://github.com/openapi-ts/openapi-typescript/assets/36955373/0464ba76-6880-4328-90ba-fb134b217e82)

Reply:
> Maybe if it really starts to be an ecosystem an org scope will be interesting.

I agree—a scope would be good, but I admit I didn’t have the foresight to register `@openapi-ts` years ago back when this was just a single package. I just reached out to the owner of `@openapi-ts` to see if they are still using it 🤞.

I’m a little hesitant to adopt the `@oapi-ts` scope I do have access to, unless we used the `oapi` abbreviation everywhere (and changed the GitHub org + domain). I have seen that abbreviation here and there, but not as much. It’s difficult having a commonly-named project—lots of potential for collision (but also I think it’s led to the adoption of this project—being very Google-able)